webapp/src/app.tsx

```diff
@@ -1760,6 +1782,15 @@ export class ProjectView
             'full-abs'
         ];
         const rootClasses = sui.cx(rootClassList);
+
+        if (this.state.hasError) {
+            return <div id="root" className="ui middle aligned center aligned grid" style={{ height: '100%', alignItems: 'center' }}>
+                <div className="ui raised segment inverted purple">
+                    <h2>{lf("Oops")}</h2>
+                    {lf("We detected a problem and we will reload the editor in a few seconds..")}
```

Comment:
Show a different message if it isn't going to reload? No idea what that message should say.

Reply:
I think that’s over complicating the feature. Users really shouldn’t get into a reload loop, we just added a bit of logic to avoid that.

Reply:
I think that’s over complicating the feature. Users really shouldn’t get into a reload loop, we just added a bit of logic to avoid the possibility